mscore/qmldockwidget.cpp

```diff
@@ -211,8 +211,10 @@ QQuickView* QmlDockWidget::getView()
 
 void QmlDockWidget::ensureQmlViewFocused()
       {
-      if (_view && !_view->activeFocusItem())
+      if (_view && !_view->activeFocusItem()) {
+            widget()->activateWindow();
```

Comment:
This ends up allowing using the search shortcut while floating. Unfortunately took quite some time for me to figure this out.